solutions/devsprint-michael-douglas-2/DeliveryApp/Screens/Components/EmptyView.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  EmptyView.swift
+//  DeliveryApp
+//
+//  Created by Sara Batista dos Santos Felix (P) on 22/02/22.
+//
+
+import UIKit
+
+final class EmptyView: UIView {
+    private lazy var emptyLabel: UILabel = {
+        let emptyTextLabel = UILabel()
+        emptyTextLabel.translatesAutoresizingMaskIntoConstraints = false
+        emptyTextLabel.text = "Endereço não encontrado"
+        emptyTextLabel.font = .boldSystemFont(ofSize: 10)
+        emptyTextLabel.textAlignment = .center
+        return emptyTextLabel
+    }()
```

Comment:
Considerando que sua classe já se chama `EmptyView`, você pode otimizar o nome das suas propriedades deixando apenas a utilidade delas.
```suggestion
    private lazy var titleLabel: UILabel = {
        let titleLabel = UILabel()
        titleLabel.translatesAutoresizingMaskIntoConstraints = false
        titleLabel.text = "Endereço não encontrado"
        titleLabel.font = .boldSystemFont(ofSize: 10)
        titleLabel.textAlignment = .center
        return titleLabel
    }()
```